README.md

```diff
@@ -86,117 +130,67 @@ This is way more readable and easier to follow along. At the same time everythin
 
 ## Installation
 
-1.  Add package
+### 1. Add package
 
 `yarn add @commercetools/enzyme-extensions -D`
 
-2.  Add a test setup file (test runner dependent)
+### 2. Add a test setup file (test runner dependent)
 
 For Jest you would set up a [`setupTestFrameworkScriptFile`](https://facebook.github.io/jest/docs/configuration.html#setuptestframeworkscriptfile-string).
 Create that file and add it to the jest configuration.
 
-3.  Add expand to Enzyme
+### 3. Add `renderProp` to Enzyme
 
 In that `testFrameworkScriptFile` file, import the extensions and add them to Enzyme
 
 ```js
-import Enzyme, { shallow } from 'enzyme';
+import Enzyme from 'enzyme';
 import Adapter from 'enzyme-adapter-react-xx';
-import { expand } from '@commercetools/enzyme-extensions';
+import { renderProp, until } from '@commercetools/enzyme-extensions';
 import ShallowWrapper from 'enzyme/ShallowWrapper';
 
+// you likely had this part already
 Enzyme.configure({ adapter: new Adapter() });
-ShallowWrapper.prototype.expand = expand;
-```
 
-## Usage
-
-#### `expand(selector, { propName?: string, props?: Object })`
-
-Expand a `ShallowWrapper` through the passed `node` and `propName` with the passed `props`. The `options` can have:
-
-1.  `propName`: the prop being the Render Prop - so `children`, `render` or any other `renderThatThingy`
-2.  `props`: the props you want to pass into the Render Prop based component
-3.  `wrapper`: defaults to `div` used to wrap the component. Often it's `null` when wanting to take a snapshot after a sequence of chained `expand`s
-
-_Note_: that any other properties on `options` than listed above are automatially forwarded to `shallow(node, options)`. This allows you to e.g. pass on a stubbed `context`.
-
-```js
-import FunctionAsAChildComponent from 'somewhere';
-
-describe('Component', () => {
-  const wrapper = shallow(<Component />);
-  const nestedWrapperProps = {
-    onChange: jest.fn(),
-    ...props,
-  };
-  let nestedWrapper;
-
-  beforeEach(() => {
-    nestedWrapper = wrapper.expand(FunctionAsAChildComponent, {
-      propName: 'render',
-      props: nestedWrapperProps,
-    });
-  });
-
-  it('should match snapshot', () => {
-    expect(nestedWrapper).toMatchSnapshot();
-  });
-
-  it('should pass `onClick` to `<button>`', () => {
-    nestedWrapper.find('button').prop('onClick')();
-
-    expect(nestedWrapperProps.onChange).toHaveBeenCalled();
-  });
-});
+// this is the actual integration
+ShallowWrapper.prototype.renderProp = renderProp;
+ShallowWrapper.prototype.until = until;
```

Comment:
I was thinking whether it's easier to export a `configure` function, which would be used like this:

```js
import configure from `@commercetools/enzyme-extensions`

configure(enzyme)
// or
configure(ShallowWrapper)
```

Reply:
That's a nice idea and should work. I would like it only working with passing in `enzyme` as people will have that imported in their configuration files already to setup a react-adpater but I don't think it'll work.

Reply:
2d5d7ef